### PR TITLE
Genezalize the kings island parser so it can parse all of adjenti's islands

### PR DESCRIPTION
**Pull request by ajoga**
This commit supports flinders island (#2533) and rottnest island (#2534), AUS (bounding_boxes are missing though, I'm gonna try to understand #2857 to see if I need to add something in this PR about that. inputs welcome in the meantime)

### CODE REVIEW

File: config/zones.json
Comment (q--) [inline config/zones.json:331]: It's spelled "ajenti"

Alternatively, you could also name this after Hydro Tasmania

Comment (ajoga) [reply config/zones.json:331]: damn me. Thanks for the catch. 

I'm going to stay with ajenti as they're the Hydro Tasmania contractor/subsidiary for these project and I think they could power projects for other operators (see [who are we ? from their website](https://adms.ajenti.com.au/Info/projects.htm))

**Review by FelixDQ — COMMENTED** [review]
Awesome @ajoga! We need to seperate the geometries. You can see how King Island was separated here https://github.com/tmrowco/electricitymap-contrib/pull/1780
Also see this readme for more information about how the world is generated https://github.com/tmrowco/electricitymap-contrib/blob/master/web/README.md


After this we can generate the bounding boxes with this script https://github.com/tmrowco/electricitymap-contrib/blob/master/web/generate-zone-bounding-boxes.js

Let me know if you have problems :-)

**Comment by chuqtas** [comment]
@FelixDQ - I've submitted a pull request for the Flinders Island geometry - #2925 

Once that is reviewed/approved I can attempt the same for Rottnest Island.

**Comment by FelixDQ** [comment]
https://github.com/tmrowco/electricitymap-contrib/pull/2925 has been merged. When we can merge this when we have to geometry for Rottnest Island as well :-)

**Comment by ajoga** [comment]
Alright I merged this parser back with master and added some details for Flinders island and fixed a now invalid key in zones.json. 

> #2925 has been merged. When we can merge this when we have the geometry for Rottnest Island as well :-)

Indeed we need #2977 to merge so we can have the the script generating the bounding box do the work for us. I'll keep an eye on it

**Comment by madsnedergaard** [comment]
Seems like we're very close to the end of this now that #2977 has been merged 🥳 
Do you need to make more changes @ajoga or is it ready for a final review?

**Comment by ajoga** [comment]
Thank you for chiming in @madsnedergaard. To me this PR can get its final review and be merged whenever you want!

File: parsers/ajenti.py
Comment (madsnedergaard) [inline parsers/ajenti.py:34]: This should be updated to make the parser work :)
```suggestion
    'AUS-WA-RI': {
```

**Review by madsnedergaard — CHANGES_REQUESTED** [review]
Great work here, one change is required to make AUS-WA-RI work - but otherwise LGTM! :)

File: parsers/ajenti.py
Comment (ajoga) [reply parsers/ajenti.py:34]: Good catch! Thanks

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**
